app/services/data/get-claim-list-for-search.js

```diff
@@ -23,8 +23,8 @@ module.exports = function (query, offset, limit) {
         .orWhere('Visitor.NationalInsuranceNumber', 'like', query)
         .orWhereRaw(`CONCAT(Visitor.FirstName, ' ', Visitor.LastName) like '${query}'`)
         .orWhere('Prisoner.PrisonNumber', 'like', query)
-        .select('Claim.Reference', 'Visitor.FirstName', 'Visitor.LastName', 'Claim.DateSubmitted', 'Claim.DateOfJourney', 'Claim.ClaimType', 'Claim.ClaimId', 'Claim.AssignedTo', 'Claim.AssignmentExpiry', 'Claim.Status')
-        .orderBy('Claim.DateSubmitted', 'asc')
+        .select('Claim.Reference', 'Visitor.FirstName', 'Visitor.LastName', 'Claim.DateSubmitted', 'Claim.DateOfJourney', 'Claim.ClaimType', 'Claim.ClaimId', 'Claim.AssignedTo', 'Claim.AssignmentExpiry', 'Claim.Status', 'Claim.LastUpdated')
+        .orderBy('Claim.LastUpdated', 'dsc')
```

Comment:
I think 'dsc' needs to be 'desc'
```suggestion
        .orderBy('Claim.LastUpdated', 'desc')
```